src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -479,8 +502,8 @@ static s32 sPrevFloorProperty = FLOOR_PROPERTY_0; // floor property from the pre
 static s32 sShapeYawToTouchedWall = 0;
 static s32 sWorldYawToTouchedWall = 0;
 static s16 sFloorShapePitch = 0;
-static s32 D_80853614 = 0;
-static s32 D_80853618 = 0;
+static s32 sUseHeldItem = 0;          // When true, the current held item is used. Is reset to false every frame.
+static s32 sHeldItemButtonIsHeld = 0; // Indicates if the controller button for the current held item is held down.
```

Comment:
```suggestion
static s32 sUseHeldItem = false;          // When true, the current held item is used. Is reset to false every frame.
static s32 sHeldItemButtonIsHeld = false; // Indicates if the controller button for the current held item is held down.
```

Reply:
I also changed `sHeldItemButtonIsHeld` -> `sHeldItemButtonIsHeldDown`
mainly because the double meaning of held here broke my brain for a second lol. hopefully this makes it more clear that the first held is for held-item and the second one means the button is physically held down